trace/templates/workflow.tmpl

```diff
@@ -0,0 +1,27 @@
+{{ indent .Depth }} {{ faint "╪" }} {{ statusIcon .State }} {{ .State.GetName }} {{ template "extras" . }}
+{{ indent .Depth 1 }} {{ with .State.Execution }}{{ faint "wfId:" }} {{ .WorkflowId |  blue }} {{- faint ", runId:" }} {{ .RunId | blue }}{{ end }}
```

Comment:
Space is tight in the terminal and workflow ids are already long enough. I would advise against using the whole word and "id" isn't descriptive by itself.

Feel free to change it if you think otherwise, though.

Reply:
We use `Id` in some of the SDKs, i think it's fine here too.